charts/jenkins/templates/jenkins-backup-cronjob.yaml

```diff
@@ -29,6 +29,8 @@ spec:
         spec:
           restartPolicy: OnFailure
           serviceAccountName: {{ template "jenkins.fullname" . }}-backup
+          securityContext:
```

Comment:
How do these lines relate to the change?

Reply:
The pod running the backup job is unable to read the token file available in the service account. It gives the error
unable to read file at /var/run/secrets/eks.amazonaws.com/serviceaccount/token: permission denied
To solve this issue, we need to set security context for this pod.

Reply:
this should probably be configurable? I'm not sure if it could break for anyone adding the security context here, thoughts @torstenwalter ?

Reply:
Hi @torstenwalter, please let me know what should be the best way. Would like to here your thoughts.

Reply:
Could we use the same security context here as for the deployment?
https://github.com/jenkinsci/helm-charts/blob/main/charts/jenkins/values.yaml#L140-L144

https://github.com/jenkinsci/helm-charts/blob/main/charts/jenkins/templates/jenkins-master-deployment.yaml#L73-L79

Reply:
Hi @torstenwalter, good point, tried with the security context of the deployment and backup job ran fine. I have changed the PR to add this security context as configurable. Included the changes in values.yaml and changelog. Please have a look. Thanks